packages/lwc/index.d.ts

```diff
@@ -207,61 +207,58 @@ declare module 'lwc' {
      * @param config configuration object for the adapter
      */
     export function wire<
-        Config extends StringKeyedRecord = any,
-        Value = any,
-        Context extends StringKeyedRecord = any
```

Comment:
Since `context` is optional, we should add a default type like `Context extends StringKeyedRecord = StringKeyedRecord`.